src/ontology/edits/oeo-physical.omn

```diff
@@ -6783,6 +6783,30 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1515",
         OEO_00010121 some OEO_00000061
     
     
+Class: OEO_00010413
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A subsurface collector is a heat exchanger that transfers ambient thermal energy to a working fluid. It consists of plastic pipes laid horizontally.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A subsurface collector is a heat exchanger that transfers ambient thermal energy to a working fluid. It consists of plastic pipes laid horizontally under the ground.",
```
When reading the def I think the "subsurface" part should somehow appear...

Reply:
I agree.